### DISCUSSION TITLE
请问有没有方法让背景色变成透明的？

### DISCUSSION THREAD

**Discussion by PuffMeow** [galacean/discussion#54] [Engine] [answered]
![1](https://user-images.githubusercontent.com/63894579/189890519-9a2e145d-2b40-4a5a-8af4-a700a10d94ef.jpg)

就是让模型后面的白底变成透明的，设置了 WebGLEngine 的 options 里的 alpha 也没生效，请问还需要做什么额外处理么？

**Comment by eyworldwide** [comment, marked as answer]
@FacesPass https://github.com/oasis-engine/engine/discussions/976
